components/operator/Makefile

```diff
@@ -102,6 +102,37 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	$(KUSTOMIZE) build config/default | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
 
+.PHONY: helm-update
+helm-update: manifests generate
```

Comment:
Maybe need to exclude Kutomiszation.yaml ?

Reply:
I did it in .helmignore and .gitignore